src/test/java/com/philips/swcoe/cerberus/DisplayHoundsTest.java

```diff
@@ -31,10 +32,14 @@ public void testRenderingHoundsOfCerberus() {
     public void testFormatRenderingHoundsOfCerberus() {
         String textTableString = getHelpStringToTest();
         String[] arrayOfHelpStringsDisplayed = textTableString.split("\n");
-        assertTrue(arrayOfHelpStringsDisplayed[0].contains("CPD            Detect duplicated blocks of code in your source code"));
-        assertTrue(arrayOfHelpStringsDisplayed[1].contains("SWD            Detect all the warnings which are suppressed in your code"));
-        assertTrue(arrayOfHelpStringsDisplayed[2].contains("JCMD           Java Code Metrics Detector"));
-        assertTrue(arrayOfHelpStringsDisplayed[3].contains("JCMD-DIFF      Java Code Metrics Detector with Diff"));
+        assertTrue(arrayOfHelpStringsDisplayed[0]
+            .contains("CPD            Detect duplicated blocks of code in your source code"));
+        assertTrue(arrayOfHelpStringsDisplayed[1]
+            .contains("SWD            Detect all the warnings which are suppressed in your code"));
+        assertTrue(
+            arrayOfHelpStringsDisplayed[2].contains("JCMD           Java Code Metrics Detector"));
+        assertTrue(arrayOfHelpStringsDisplayed[3]
+            .contains("JCMD-DIFF      Java Code Metrics Detector with Diff"));
     }
```

Comment:
Worth considering Hamcrest's assertThat("", containsString(""));? 
http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/Matchers.html

Reply:
Thanks for sharing this, I have a next Test refactor iteration planned in another branch will consider and fix it on that branch and create another PR